doc/api/errors.md

```diff
@@ -253,6 +256,45 @@ will affect any stack trace captured _after_ the value has been changed.
 If set to a non-number value, or set to a negative number, stack traces will
 not capture any frames.
 
+### `error.cause`
+
+<!-- YAML
+added: v16.9.0
+-->
+
+* {any}
+
+The `error.cause` property is the cause of the error, as set by calling
+`new Error(message, { cause })`. It is not set if the `cause` option is not
+provided.
```

Comment:
I wonder if properties that are set or not is obvious to all JS devs, so maybe instead we could with something simpler:

```suggestion
Not all errors have a `cause` property.
```

Reply:
IMO, I will prefer the original description. "Not all errors have a `cause` property." is very obvious, I think we don't need to add this sentence.

Reply:
I kept most of the original wording but changed the first paragraph to start with, "If present,  the `error.cause` property..." similar to how other optional properties are described on this page.